utils/certs.go

```diff
@@ -93,7 +93,7 @@ func GenerateCert(hosts []string, certFile, keyFile, caFile, caKeyFile, org stri
 	}
 	// client
 	if len(hosts) == 1 && hosts[0] == "" {
-		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth}
+		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth}
```

Comment:
Correct, basically whatever address the user is using to connect to the swarm master.
This is why I was thinking about the "none" driver, which would let someone create a "machine" but for their swarm master... But I suppose that is a hack.